### DISCUSSION TITLE
Materials System shared among the other systems

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#27963] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

All,
It reads that the Materials System allows properties to be defined in a single object (a Material) and shared among the many other systems such as the Kernel or BoundaryCondition Systems.

I was trying to share the values obtained in the Materials System with an object in the InterfaceKernels System and seemed not to work.  Can the Materials System share with only Kernel System and BoundaryCondition System?
Please help. S. Thomas

**Comment by GiudGiud** [comment, marked as answer]
Hello

Interface kernels have slightly different routine names for retrieving material properties, notably because they need to retrieve both the element and neighbor material properties 

Look for examples among existing interface kernels
